src/semantics.c

```diff
@@ -721,6 +721,9 @@ void parse_declarator(type_element modifiers, declarator d, bool bitfield,
 		    break;
 		  }
 		/* Fall through */
+              case RID_NORETURN:
```

Comment:
To a  first approximation, _Noreturn should be handled like inline (from a "checking use" perspective, their spec is identical in C11). So:
- no need for a special case here, it should just go to the default like inline
- parse_scflags and check_variable_scflags above should handle _Noreturn like inline (this will cause nesC to correctly report an error if, e.g., _Noreturn is used on a variable declaration) [I'd put this comment above, but github won't let me]